### PR TITLE
Add CodeFuse queries as example

### PR DESCRIPTION
**Pull request by FunJim**

### CODE REVIEW

File: example/CodeFuse/CommentRatioJava.gdl [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+                     parent_hash_id : tmp.parent_hash_id,
+                     location_hash_id : tmp.location_hash_id,
+                     definition_body : tmp.definition_body
+                 }
+              }
+           }
+        }
+    }
+    
+
```
Comment (gangfan) [inline example/CodeFuse/CommentRatioJava.gdl:247]: extra spaces.

Comment (FunJim) [reply example/CodeFuse/CommentRatioJava.gdl]: fixed

File: example/CodeFuse/CommentCodePairsJava.gdl [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+            }
+        }
+    }
+}
+
+//找到Javadoc注释所在文件和行号信息
+pub fn getJavadocCommentLocInfo(e: JavadocComment, i: int, j: int) -> bool {
+    let (db = default_db()) {
+        for (n in int::__undetermined_all__()) {
+                    if (i = e.getLocation().getFile().element_hash_id) {
```
Comment (ValKmjolnir) [inline example/CodeFuse/CommentCodePairsJava.gdl:207]: indentation in this line is not correct, make sure you are using 4 spaces instead of tab

**Review by ValKmjolnir — CHANGES_REQUESTED** [review]
Too many indentation related issues, make sure using 4 spaces instead of tab

File: example/CodeFuse/CommentCodePairsJava.gdl [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+            }
+        }
+    }
+}
+
+//过滤得到的自动生成文件名
+pub fn getAutoGeneratedFiles(n: string) -> bool {
+    let (db = default_db()) {
+        for (f in File(db)) {
+        if (n = f.getRelativePath()) {
```
[the remark this answers is not in the document]
Comment (FunJim) [reply example/CodeFuse/CommentCodePairsJava.gdl]: should be all fixed.

**Review by ValKmjolnir — APPROVED** [review]
LGTM

**Pull request merged**